src/v2/Components/RouteTabs.tsx

```diff
@@ -33,16 +33,16 @@ export const RouteTabs = styled(TabsContainer)`
   }
 `
 
-export const RouteTab: React.FC<Omit<LinkProps, "ref">> = ({
+export const RouteTab: React.FC<Omit<RouterLinkProps, "ref">> = ({
   children,
   ...props
 }) => {
   return (
-    <Link {...props} activeClassName="active">
+    <RouterLink {...props} activeClassName="active">
       <Sans size="3t" weight="medium">
         {children}
       </Sans>
-    </Link>
+    </RouterLink>
```

Comment:
@damassi ☝️  — switched this to use `RouterLink` so that the specs pass, but lemme know if you'd prefer otherwise.

Reply:
Interesting, what was the problem in terms of specs? Like why suddenly failing? 

Eitther way, it _should_ be a router link. We have [a mechanism](https://github.com/artsy/force/blob/master/src/v2/Artsy/Router/Utils/catchLinks.tsx) to catch regular links to redirect internally tot the router, but thats mostly for older content. (Also noticed some issues with hot reloading during dev when using regular links over the RouterLink, so whenever its encountered it should be replaced -- assuming we're linking to new pages of course.)